packages/api/app/Models/Traits/InvalidateCache.js

```diff
@@ -0,0 +1,19 @@
+const { cache } = require('../../Utils');
+
+const deleteKeysFromPrefix = async (prefix) => {
+	const keys = await cache.keys(`${prefix}*`);
+
+	for (const key of keys) {
+		// eslint-disable-next-line no-await-in-loop
+		await cache.del(key);
+	}
```

Comment:
Search for all redis keys by prefix and delete one by one.
Unfortunately, redis does not have a native function for delete in bulk.